### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by abhinav**
This runs a `glide up` on the repository, updates the generated code, and fixes
a minor breakage due to slightly better error messages in `gomock`.

### CODE REVIEW

**Comment by breerly** [comment]
Suggest axing Travis once and for all.

Also, lets get the build passing :)

**Comment by breerly** [comment]
Aaah, `grpc.WithCodec` has been marked as deprecated:

``` 

transport/grpc/peer.go:50:3: grpc.WithCodec is deprecated: use WithDefaultCallOptions(CallCustomCodec(c)) instead.  (SA1019)
--
  | etc/make/local.mk:88: recipe for target 'staticcheck' failed
  | make: *** [staticcheck] Error 1

```

File: encoding/thrift/thriftrw-plugin-yarpc/gomock_test.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func TestMockClients(t *testing.T) {
 			},
 			wantStatus: Fatal,
 			wantErrorsLike: []string{
-				"missing call(s) to [*extendemptytest.MockClient.Hello(is anything)]",
+				"missing call(s) to [*extendemptytest.MockClient.Hello(is anything)",
```
[the remark this answers is not in the document]
Comment (abhinav) [reply encoding/thrift/thriftrw-plugin-yarpc/gomock_test.go]: going to update the commit message for that change with an explanation

**Comment by peter-edge** [comment]
Heh I think that was me. It’s a quick fix in transport/grpc.

File: glide.lock [hunk 2]
```diff
@@ -136,15 +136,15 @@ imports:
   - internal/fxreflect
   - internal/lifecycle
 - name: go.uber.org/multierr
-  version: 3c4937480c32f4c13a875a1829af76c98ca3d40a
+  version: a3d1fc1f1316d4132fc61f4ea1159ae0613fb474
```
[the remark this answers is not in the document]
Comment (abhinav) [reply glide.lock]: Good catch. I'll let the other PR fix this.

**Pull request merged**